src/content/plugins/split-chunks-plugin.md

```diff
@@ -205,6 +205,30 @@ A module can belong to multiple cache groups. The optimization will prefer the c
 
 If the current chunk contains modules already split out from the main bundle, it will be reused instead of a new one being generated. This can impact the resulting file name of the chunk.
 
+#### `splitChunks.cacheGroups.type`
+
+`function` `RegExp` `string`
+
+Allows to assign modules to a cache group by module type.
+
+__webpack.config.js__
+
+```js
+module.exports = {
+  //...
+  optimization: {
+    splitChunks: {
+      cacheGroups: {
+        json: {
+          name: 'json',
```

Comment:
I think it would be nice to change `json` to something like `json-chunk` or similar just to distinguish the name from the type below and make it less confusing?

Reply:
name shouldnt be there, was copying code from the autotest in webpack repo. i will remove